From 238680060c9ab66f87b2d4f4bddc033083b2af4d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
 <congdanhqx@gmail.com>
Date: Mon, 2 Nov 2020 07:17:07 +0700
Subject: [PATCH] polyglot: don't try to import plistlib.Data

plistlib.Data is deprecated from Python 3.4 and fully removed in Python
3.9.

Skimming over the code, it's look like nothing uses this data structure.

Let's remove it.
---
 src/polyglot/plistlib.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git src/polyglot/plistlib.py src/polyglot/plistlib.py
index e1a5b8b16a..c55811e909 100644
--- src/polyglot/plistlib.py
+++ src/polyglot/plistlib.py
@@ -2,5 +2,5 @@
 # vim:fileencoding=utf-8
 # License: GPL v3 Copyright: 2019, Kovid Goyal <kovid at kovidgoyal.net>
 
-from plistlib import loads, dumps, Data  # noqa
+from plistlib import loads, dumps  # noqa
 loads_binary_or_xml = loads
